ldelf/ta_elf.c

```diff
@@ -855,6 +861,56 @@ static void populate_segments(struct ta_elf *elf)
 	}
 }
 
+static void parse_property_segment(struct ta_elf *elf)
+{
+	char *desc = NULL;
+	size_t align = elf->prop_align;
+	size_t desc_offset = 0;
+	size_t prop_offset = 0;
+	vaddr_t va = elf->load_addr + elf->prop_start;
```

Comment:
ADD_OVERFLOW() ?

Reply:
I have already taken care of this in -https://github.com/OP-TEE/optee_os/pull/4967/commits/672a4e8f8fb1f20a415ec3f2e35fa6c41154c811